src/flow/domain/single.rs

```diff
@@ -224,8 +224,11 @@ impl NodeDescriptor {
                 let from = m.link().src;
 
                 let replay = if let Packet::ReplayPiece {
-                           context: flow::payload::ReplayPieceContext::Partial { ref for_key }, ..
-                       } = *m {
+                    context: flow::payload::ReplayPieceContext::Partial {
+                        ref for_key, ignore
```

Comment:
Actually, the more I think about it, the more I think `ignore` should be replaced with a `ReplayPieceContext` variant like `TransactionalIgnore`.